__tests__/infrastructure/CreateTransaction.spec.js

```diff
@@ -113,4 +119,164 @@ describe('CreateTransaction should', () => {
 			namespaceName: [namespace]
 		});
 	});
+
+	it('returns transferTransaction', async () =>  {
```

Comment:
while this is true statment, test is really verifying that address and mosaics within transferTransaction are resolved, so something like 

resolves mosaics and addresses in transfer transaction 

would be more descriptive around intent of test imo

Reply:
make sense, i will update the description 👍🏼